src/App.tsx

```diff
@@ -1,7 +1,8 @@
+import { lazy } from 'react';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
-import Home from './pages/Home/Home';
-import Search from './pages/Search/Search';
+const Search = lazy(() => import('./pages/Search/Search'));
+const Home = lazy(() => import('./pages/Home/Home'));
```

Comment:
```suggestion
const Home = lazy(() => import(/* webpackChunkName: "home" */ './pages/Home/Home'));
const Search = lazy(() => import(/* webpackChunkName: "search" */ './pages/Search/Search'));
```

이런식으로 주석을 추가하면 webpack 번들링 과정에서 나온 청크파일에 작성한 이름을 부여한다고 하네요. 대신 사용하려면 tsconfig 파일에서 컴파일 과정 중 주석 삭제를 하지 않도록 해야합니다!

Reply:
오! 배워갑니다👍